app/client/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,42 @@
+import { Grid, Button } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        display: 'flex',
+        padding: theme.spacing(1),
+        background: '#DDDDDD',
+        borderRadius: '7px',
```

Comment:
Please use theme based colors.  I think there should be one for grey.  If not you can add it.